action.yml

```
@@ -28,7 +28,10 @@ inputs:
       Can be a specific version from npm (e.g., '0.1.0', 'latest'), a branch name (e.g., 'main'), or a commit hash.
     required: false
     default: 'latest'
-
+outputs:
+  gemini_cli_results:
```

please rename this to summary
```suggestion
  summary:
```

---

and the step below needs to also be updated